src/main/java/uk/gov/hmcts/darts/common/util/DateConverters.java

```diff
@@ -21,4 +21,8 @@ public OffsetDateTime offsetDateTimeFrom(final String timeString) {
               .toOffsetDateTime();
     }
 
+    public LocalDateTime offsetDateTimeToLegacyDateTime(final OffsetDateTime offsetDateTime) {
+        return offsetDateTime.atZoneSameInstant(ASSUMED_SOURCE_ZONE_ID).toLocalDateTime();
```

Comment:
I was looking at something similar today actually for the audio download zip file that contains a readme / annotations under darts-api.

I think it would be safer to do something similar in darts-gateway actually like

`ZonedDateTime localEventTimestamp = event.getTimestamp().atZoneSameInstant(EUROPE_LONDON_ZONE);`

https://github.com/hmcts/darts-api/blob/DMP-708-ATSDownloadZipFile/src/main/java/uk/gov/hmcts/darts/audio/component/impl/AnnotationXmlGeneratorImpl.java#L77

It looks strange but the reason being that the system clock on the server is expected to be in `UTC`. So LocalDateTime will not be suitable. I came to the conclusion today that using `ZonedDataTime` to present date and time to users correctly in BST.  To store data and time, we use `OffsetDateTime` in UTC and to present date and time to users, we should use the `ZonedDataTime` for the legacy interfaces.  According to the restful api standards for the darts-api we expect request / response for date-time strings in JSON to be in UTC.

Reply:
I've done something similar now in darts-api - https://github.com/hmcts/darts-api/pull/335#discussion_r1294913443